Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2004-08-26
Initial Package Version: 2.6.8
Origin: Jim Gifford
Upstream Status: Sent
Description: Installs Man-Pages
 
diff -Naur iproute2-2.6.8.orig/Makefile iproute2-2.6.8/Makefile
--- iproute2-2.6.8.orig/Makefile	2004-08-23 20:22:16.000000000 +0000
+++ iproute2-2.6.8/Makefile	2004-08-26 22:19:44.217544872 +0000
@@ -2,6 +2,7 @@
 SBINDIR=/usr/sbin
 CONFDIR=/etc/iproute2
 DOCDIR=/usr/doc/iproute2
+MANDIR=/usr/share/man
 
 # Path to db_185.h include
 DBM_INCLUDE:=/usr/include
@@ -46,7 +47,9 @@
 		$(DESTDIR)$(DOCDIR)/examples/diffserv
 	@for i in $(SUBDIRS) doc; do $(MAKE) -C $$i install; done
 	install -m 0644 $(shell find etc/iproute2 -type f -maxdepth 1) $(DESTDIR)$(CONFDIR)
-
+	install -m 0644 $(shell find man/man8 -type f -maxdepth 1) $(DESTDIR)$(MANDIR)/man8
+	ln -sf $(DESTDIR)$(MANDIR)/man8/tc-pbfifo.8  $(DESTDIR)$(MANDIR)/man8/tc-bfifo.8
+	ln -sf $(DESTDIR)$(MANDIR)/man8/tc-pbfifo.8  $(DESTDIR)$(MANDIR)/man8/tc-pfifo.8
 
 clean:
 	@for i in $(SUBDIRS) doc; \
diff -Naur iproute2-2.6.8.orig/man/man8/tc-bfifo.8 iproute2-2.6.8/man/man8/tc-bfifo.8
--- iproute2-2.6.8.orig/man/man8/tc-bfifo.8	2004-08-23 20:22:16.000000000 +0000
+++ iproute2-2.6.8/man/man8/tc-bfifo.8	1970-01-01 00:00:00.000000000 +0000
@@ -1,72 +0,0 @@
-.TH PBFIFO 8 "10 January 2002" "iproute2" "Linux"
-.SH NAME
-pfifo \- Packet limited First In, First Out queue
-.P
-bfifo \- Byte limited First In, First Out queue
-
-.SH SYNOPSIS
-.B tc qdisc ... add pfifo
-.B [ limit 
-packets
-.B ]
-.P
-.B tc qdisc ... add bfifo
-.B [ limit 
-bytes
-.B ]
-
-.SH DESCRIPTION
-The pfifo and bfifo qdiscs are unadorned First In, First Out queues. They are the
-simplest queues possible and therefore have no overhead. 
-.B pfifo
-constrains the queue size as measured in packets. 
-.B bfifo
-does so as measured in bytes.
-
-Like all non-default qdiscs, they maintain statistics. This might be a reason to prefer 
-pfifo or bfifo over the default.
-
-.SH ALGORITHM
-A list of packets is maintained, when a packet is enqueued it gets inserted at the tail of
-a list. When a packet needs to be sent out to the network, it is taken from the head of the list. 
-
-If the list is too long, no further packets are allowed on. This is called 'tail drop'.
-
-.SH PARAMETERS
-.TP 
-limit
-Maximum queue size. Specified in bytes for bfifo, in packets for pfifo. For pfifo, defaults 
-to the interface txqueuelen, as specified with 
-.BR ifconfig (8)
-or
-.BR ip (8).
-
-For bfifo, it defaults to the txqueuelen multiplied by the interface MTU.
-
-.SH OUTPUT
-The output of 
-.B tc -s qdisc ls
-contains the limit, either in packets or in bytes, and the number of bytes 
-and packets actually sent. An unsent and dropped packet only appears between braces 
-and is not counted as 'Sent'.
-
-In this example, the queue length is 100 packets, 45894 bytes were sent over 681 packets. 
-No packets were dropped, and as the pfifo queue does not slow down packets, there were also no
-overlimits:
-.P
-.nf
-# tc -s qdisc ls dev eth0 
-qdisc pfifo 8001: dev eth0 limit 100p
- Sent 45894 bytes 681 pkts (dropped 0, overlimits 0) 
-.fi
-
-If a backlog occurs, this is displayed as well.
-.SH SEE ALSO
-.BR tc (8)
-
-.SH AUTHORS
-Alexey N. Kuznetsov, <kuznet@ms2.inr.ac.ru>
-
-This manpage maintained by bert hubert <ahu@ds9a.nl>
-
-
diff -Naur iproute2-2.6.8.orig/man/man8/tc-pfifo.8 iproute2-2.6.8/man/man8/tc-pfifo.8
--- iproute2-2.6.8.orig/man/man8/tc-pfifo.8	2004-08-23 20:22:16.000000000 +0000
+++ iproute2-2.6.8/man/man8/tc-pfifo.8	1970-01-01 00:00:00.000000000 +0000
@@ -1,72 +0,0 @@
-.TH PBFIFO 8 "10 January 2002" "iproute2" "Linux"
-.SH NAME
-pfifo \- Packet limited First In, First Out queue
-.P
-bfifo \- Byte limited First In, First Out queue
-
-.SH SYNOPSIS
-.B tc qdisc ... add pfifo
-.B [ limit 
-packets
-.B ]
-.P
-.B tc qdisc ... add bfifo
-.B [ limit 
-bytes
-.B ]
-
-.SH DESCRIPTION
-The pfifo and bfifo qdiscs are unadorned First In, First Out queues. They are the
-simplest queues possible and therefore have no overhead. 
-.B pfifo
-constrains the queue size as measured in packets. 
-.B bfifo
-does so as measured in bytes.
-
-Like all non-default qdiscs, they maintain statistics. This might be a reason to prefer 
-pfifo or bfifo over the default.
-
-.SH ALGORITHM
-A list of packets is maintained, when a packet is enqueued it gets inserted at the tail of
-a list. When a packet needs to be sent out to the network, it is taken from the head of the list. 
-
-If the list is too long, no further packets are allowed on. This is called 'tail drop'.
-
-.SH PARAMETERS
-.TP 
-limit
-Maximum queue size. Specified in bytes for bfifo, in packets for pfifo. For pfifo, defaults 
-to the interface txqueuelen, as specified with 
-.BR ifconfig (8)
-or
-.BR ip (8).
-
-For bfifo, it defaults to the txqueuelen multiplied by the interface MTU.
-
-.SH OUTPUT
-The output of 
-.B tc -s qdisc ls
-contains the limit, either in packets or in bytes, and the number of bytes 
-and packets actually sent. An unsent and dropped packet only appears between braces 
-and is not counted as 'Sent'.
-
-In this example, the queue length is 100 packets, 45894 bytes were sent over 681 packets. 
-No packets were dropped, and as the pfifo queue does not slow down packets, there were also no
-overlimits:
-.P
-.nf
-# tc -s qdisc ls dev eth0 
-qdisc pfifo 8001: dev eth0 limit 100p
- Sent 45894 bytes 681 pkts (dropped 0, overlimits 0) 
-.fi
-
-If a backlog occurs, this is displayed as well.
-.SH SEE ALSO
-.BR tc (8)
-
-.SH AUTHORS
-Alexey N. Kuznetsov, <kuznet@ms2.inr.ac.ru>
-
-This manpage maintained by bert hubert <ahu@ds9a.nl>
-
-
